src/components/TitleTransferPanel/AddressInput/AddressInput.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { ReactElement } from "react";
+import css from "./AddressInput.module.scss";
+
+interface AddressInputProps {
+  errorMessage?: string;
+  layoutCtaRight?: ReactElement;
+  layoutCtaBottom?: ReactElement;
+  address: string;
+  setAddress: (e: any) => void;
+  disabled?: boolean;
+}
+
+export const AddressInput = ({
+  errorMessage = "",
+  layoutCtaRight,
```

Comment:
Rather than passing in a component as a prop, which is really weird, you can:

Have the component as the child and then have a prop to determine the position

Example of the use will be:

For default on the right
```
<AddressInput {...props}>
  <AddressCta
            buttonText="Confirm"
            onButtonSubmit={() => {}}
            holderAddress="0x28F7aB32C521D13F2E6980d072Ca7CA493020145"
            beneficiaryAddress="0x48399Fb88bcD031C556F53e9303CaF7aB32C9302"
            isDisabledAddressBook={true}
          />
</AddressInput>
```

and for bottom
```
<AddressInput buttonsPosition="bottom" {...props}>
  <AddressCta
            buttonText="Confirm"
            onButtonSubmit={() => {}}
            holderAddress="0x28F7aB32C521D13F2E6980d072Ca7CA493020145"
            beneficiaryAddress="0x48399Fb88bcD031C556F53e9303CaF7aB32C9302"
            isDisabledAddressBook={true}
          />
</AddressInput>
```